src/crate/crash/command.py

```diff
@@ -499,5 +503,10 @@ def _create_cmd(crate_hosts, error_trace, output_writer, is_tty, args):
                     ca_cert_file=args.ca_cert_file,
                     username=args.username)
 
+def file_with_permissions(path):
+    if not os.access(path, os.R_OK):
+        raise Exception('[' + path + '] does not exist or is not readable')
```

Comment:
What kind of Exception? Should I create a new class like ValidationError or ParseError?